### ISSUE TITLE
ApolloServer config mentions "cors" but it's not included in the type definition

### ISSUE THREAD

**Issue by kevinburke** [apollographql/apollo-server#1882]
- [X] docs

   > Documentation issues should be labeled, but if you know the answer
   > please consider submitting a Pull Request instead.  (The documentation
   > lives in the `docs/source` directory of this repository.)

The documentation for Apollo Server mentions it can accept `cors` as an argument to the ApolloServer constructor: https://www.apollographql.com/docs/apollo-server/api/apollo-server.html

```
cors: <Object | boolean> (apollo-server)

Pass the integration-specific CORS options. false removes the CORS middleware and true uses the defaults. This option is only available to apollo-server. For other server integrations, place cors inside of applyMiddleware.
```

However, the type definition for Config does not include a cors object:

```
export interface Config extends Pick<GraphQLOptions<Context<any>>, 'formatError' | 'debug' | 'rootValue' | 'validationRules' | 'formatResponse' | 'fieldResolver' | 'cacheControl' | 'tracing' | 'dataSources' | 'cache'> {
    typeDefs?: DocumentNode | Array<DocumentNode>;
    resolvers?: IResolvers;
    schema?: GraphQLSchema;
    schemaDirectives?: Record<string, typeof SchemaDirectiveVisitor>;
    context?: Context<any> | ContextFunction<any>;
    introspection?: boolean;
    mocks?: boolean | IMocks;
    mockEntireSchema?: boolean;
    engine?: boolean | EngineReportingOptions;
    extensions?: Array<() => GraphQLExtension>;
    persistedQueries?: PersistedQueryOptions | false;
    subscriptions?: Partial<SubscriptionServerOptions> | string | false;
    uploads?: boolean | FileUploadOptions;
    playground?: PlaygroundConfig;
}
```

Running apollo-server-core v2.1.0.

**Comment by Can2Nya** [comment]
some problem,mark

**Comment by abernix** [comment]
This should be fixed thanks to @cheapsteak's #2330 and #2373.  Thanks for reporting this!
